package.json

```diff
@@ -15,7 +15,7 @@
         "webpack-watch": "webpack --watch --progress --color",
         "lint": "eslint src/ --ext=.ts,.tsx",
         "lint:fix": "eslint --fix src/ --ext=.ts,.tsx",
-        "test": "jest",
+        "test": "`jest`",
```

Comment:
Why?

Reply:
Whoa, what is that?   No idea - it looks like something a prettier might have done???